webdriver/tests/print/printcmd.py

```diff
@@ -1,21 +1,17 @@
-import base64
+from base64 import decodebytes
 
 import pytest
 
-import six
-
 from tests.support.asserts import assert_error, assert_success
 
-def decodebytes(s):
-    return base64.decodebytes(six.ensure_binary(s))
 
-def do_print(session, options):
+def do_print(session, options) -> str:
```

Comment:
I think this would be the first type annotation in the code base not using the `# type: ...` comments. @gsnedders @jgraham do you know of anything bad that's going to result from this?

Reply:
Only that the value of type annotations in code that we don't actually type check is equivalent to that of a comment in the sense that it's just as likely to be wrong, but also looks more authoritative. 

I think if we want to start adding type annotations we should also figure out how we're going to check them (and this is perhaps not a small problem for test files because it likely means vendors should be running the exact same version of mypy in their CI; for tools changes it's kind of assumed that people submitting patches to vendor repos are able to deal with the consequences of failing upstream checks, but that is less true for test authors).

Reply:
I've filed https://github.com/web-platform-tests/wpt/issues/28833, that's the precise concern I was expecting. I'm going to back out the type annotations from this PR.